src/infra/core/src/remote_repository_registry_impl.rs

```diff
@@ -36,6 +36,23 @@ impl RemoteRepositoryRegistryImpl {
         std::fs::create_dir_all(&repos_dir)?;
         Ok(Self::new(repos_dir))
     }
+
+    pub fn get_repository_file_path(&self, repo_name: &RepoName) -> Option<PathBuf> {
+        let file_path = self.repos_dir.join(repo_name);
+
+        if !file_path.exists() {
+            // run full scan to support case-insensetive matches
+            let all_repositories_stream = self.get_all_repositories();
```

Comment:
Minor: perhaps we could disable this part on Windows, since its file system already is case-insensitive.

Reply:
If I'm not mistaken it can be set up to be a case sensitive. Not sure it is safe to disable it

Reply:
This was never possible before, but yes, I see they've added some capabilities in W10 and beyond to interract with WSL